diff -ur pdp++-2.01.orig/pdp++/src/css/css_builtin.cc pdp++-2.01/pdp++/src/css/css_builtin.cc
--- pdp++-2.01.orig/pdp++/src/css/css_builtin.cc	Fri Oct 13 16:06:12 2000
+++ pdp++-2.01/pdp++/src/css/css_builtin.cc	Fri Oct 13 17:18:30 2000
@@ -2222,7 +2222,7 @@
   taMisc::include_paths.AddUnique(home); // this is first on the list..
 
 #ifndef CYGWIN
-  String css_dir = "/usr/local/pdp++"; // default css home directory
+  String css_dir = PDPDESTDIR; // default css home directory
 #else
   String css_dir = "C:/PDP++"; // default pdp home directory
 #endif
diff -ur pdp++-2.01.orig/pdp++/src/pdp/pdpshell.cc pdp++-2.01/pdp++/src/pdp/pdpshell.cc
--- pdp++-2.01.orig/pdp++/src/pdp/pdpshell.cc	Wed Aug  9 19:18:17 2000
+++ pdp++-2.01/pdp++/src/pdp/pdpshell.cc	Fri Oct 13 17:18:07 2000
@@ -1837,7 +1837,7 @@
   taMisc::Register_Cleanup((SIGNAL_PROC_FUN_TYPE) SaveRecoverFile);
 
 #ifndef CYGWIN
-  String pdp_dir = "/usr/local/pdp++"; // default pdp home directory
+  String pdp_dir = PDPDESTDIR; // default pdp home directory
 #else
   String pdp_dir = "C:/PDP++"; // default pdp home directory
 #endif
