--- pdp++-2.0a/pdp++/src/bp/Makefile.in~	Fri Nov 10 00:48:30 1995
+++ pdp++-2.0a/pdp++/src/bp/Makefile.in	Sun Apr 30 19:47:04 2000
@@ -15,7 +15,8 @@
 #OPT_FLAG =
 #endif
 
-LIBS = -lpdp -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -ltermcap\
+LIBS = -lpdp -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -lncurses \
+	-L../../../../interviews/lib-g++/LINUX \
 	 -ltastring $(IV_LIB) -lX11 $(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
 
 PROJNAME = bp
--- pdp++-2.0a/pdp++/src/bpso/Makefile.in~	Fri Nov 10 00:49:17 1995
+++ pdp++-2.0a/pdp++/src/bpso/Makefile.in	Sun Apr 30 19:48:10 2000
@@ -16,7 +16,8 @@
 # endif
 
 LIBS = -lpdp_bp -lpdp_so -lpdp -ltamisc -livgraphic -lcss -ltypea -livmisc \
-	 -lreadline -ltermcap -ltastring $(IV_LIB) -lX11 \
+	-L../../../../interviews/lib-g++/LINUX \
+	 -lreadline -lncurses -ltastring $(IV_LIB) -lX11 \
 	$(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
 
 PROJNAME = bpso
--- pdp++-2.0a/pdp++/src/cs/Makefile.in~	Fri Nov 10 00:49:02 1995
+++ pdp++-2.0a/pdp++/src/cs/Makefile.in	Sun Apr 30 19:48:42 2000
@@ -16,7 +16,8 @@
 # endif
 
 
-LIBS = -lpdp -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -ltermcap\
+LIBS = -lpdp -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -lncurses \
+	-L../../../../interviews/lib-g++/LINUX \
 	 -ltastring $(IV_LIB) -lX11 $(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
 
 PROJNAME = cs
--- pdp++-2.0a/pdp++/src/leabra/Makefile.in~	Sat Nov 22 01:19:33 1997
+++ pdp++-2.0a/pdp++/src/leabra/Makefile.in	Sun Apr 30 19:49:39 2000
@@ -13,7 +13,8 @@
 AR_CMD = $(SIM_NONSHARED_AR_CMD)
 LIB_EXT = $(SIM_NONSHARED_LIB_EXT)
 
-LIBS = -lpdp -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -ltermcap\
+LIBS = -lpdp -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -lncurses \
+	-L../../../../interviews/lib-g++/LINUX \
 	 -ltastring $(IV_LIB) -lX11 $(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
 
 PROJNAME = leabra
--- pdp++-1.3/pdp++/src/pdp/Makefile.in~	Mon May  1 01:57:37 2000
+++ pdp++-1.3/pdp++/src/pdp/Makefile.in	Mon May  1 04:39:43 2000
@@ -18,10 +18,12 @@
 IVLIB = -lIV
 IVDEBUGLIB = -lIVd
 
-LIBS = -lpdp_bp -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -ltermcap\
+LIBS = -lpdp_bp -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -lncurses \
+	-L../../../../interviews/lib-g++/LINUX \
 	 -ltastring $(IVLIB) -lX11 $(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
 
-EXTERNAL_LIBS = -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -ltermcap\
+EXTERNAL_LIBS = -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -lncurses \
+	-L../../../../interviews/lib-g++/LINUX \
   	        -ltastring $(IVLIB) -lX11 $(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
 PROJNAME = pdp
 PROJDIR = pdp
--- pdp++-2.0a/pdp++/src/so/Makefile.in~	Fri Nov 10 00:48:47 1995
+++ pdp++-2.0a/pdp++/src/so/Makefile.in	Sun Apr 30 19:50:45 2000
@@ -15,7 +15,8 @@
 # OPT_FLAG =
 # endif
 
-LIBS = -lpdp -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -ltermcap\
+LIBS = -lpdp -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -lncurses \
+	-L../../../../interviews/lib-g++/LINUX \
 	 -ltastring -lIV -lX11 $(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
 
 PROJNAME = so
--- pdp++-2.0a/pdp++/src/ta_misc/Makefile.in~	Tue Nov  5 23:57:17 1996
+++ pdp++-2.0a/pdp++/src/ta_misc/Makefile.in	Sun Apr 30 19:51:15 2000
@@ -7,7 +7,8 @@
 # reset opt flags to not do anything for linux, crashes with dynamic linking
 #OPT_FLAGS=
 
-LIBS =  -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -ltermcap -ltastring\
+LIBS =  -ltamisc -livgraphic -lcss -ltypea -livmisc -lreadline -lncurses -ltastring\
+	-L../../../../interviews/lib-g++/LINUX \
 	$(IV_LIB) -lX11 $(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
 
 PROJNAME = ta_misc
--- pdp++-2.0a/pdp++/src/maketa/Makefile.in~	Wed Feb 25 19:00:20 1998
+++ pdp++-2.0a/pdp++/src/maketa/Makefile.in	Sun Apr 30 19:52:24 2000
@@ -8,7 +8,7 @@
 #OPT_FLAGS = -O
 
 LIBS = -ltastring $(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
-EXTERNAL_LIBS = $(LIBS) -lcss -ltamisc -livmisc -livgraphic -lreadline -ltermcap -lIV -lX11
+EXTERNAL_LIBS = $(LIBS) -lcss -ltamisc -livmisc -livgraphic -lreadline -lncurses -lIV -lX11
 PROJNAME = maketa
 PROJDIR	 = maketa
 
--- pdp++-2.0a/pdp++/src/ta/Makefile.in~	Wed Nov  6 00:42:47 1996
+++ pdp++-2.0a/pdp++/src/ta/Makefile.in	Sun Apr 30 19:53:15 2000
@@ -8,7 +8,7 @@
 #OPT_FLAGS = -O
 
 LIBS = -ltastring $(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
-EXTERNAL_LIBS = $(LIBS) -lcss -ltamisc -livmisc -livgraphic -lreadline -ltermcap -lIV -lX11
+EXTERNAL_LIBS = $(LIBS) -lcss -ltamisc -livmisc -livgraphic -lreadline -lncurses -lIV -lX11
 PROJNAME = ta
 PROJDIR	 = ta
 
--- pdp++-2.0a/pdp++/src/css/Makefile.in~	Fri May 15 23:31:45 1998
+++ pdp++-2.0a/pdp++/src/css/Makefile.in	Sun Apr 30 18:28:01 2000
@@ -7,7 +7,7 @@
 # reset opt flags to not do anything for linux, crashes with dynamic linking
 #OPT_FLAGS=
 
-LIBS = -ltamisc -livgraphic -ltypea -livmisc -lreadline -ltermcap -ltastring \
+LIBS = -ltamisc -livgraphic -ltypea -livmisc -lreadline -lncurses -ltastring \
 	$(IV_LIB) -lX11 $(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
 
 PROJNAME = css
--- pdp++-2.0a/pdp++/src/css/Makefile.in~	Sun Apr 30 18:51:09 2000
+++ pdp++-2.0a/pdp++/src/css/Makefile.in	Sun Apr 30 19:18:49 2000
@@ -8,6 +8,7 @@
 #OPT_FLAGS=
 
 LIBS = -ltamisc -livgraphic -ltypea -livmisc -lreadline -lncurses -ltastring \
+	-L../../../../interviews/lib-g++/LINUX \
 	$(IV_LIB) -lX11 $(IOS_LIB) $(IBERTY_LIB) $(GENERIC_LIB)
 
 PROJNAME = css
