--- pdp++-2.01/iv-pdp-r2.0/src/lib/IV-2_6/rubrect.cpp~	Mon Mar 13 18:11:55 2000
+++ pdp++-2.01/iv-pdp-r2.0/src/lib/IV-2_6/rubrect.cpp	Mon Sep 24 18:22:43 2001
@@ -31,11 +31,8 @@
 #include <OS/math.h>
 #include <math.h>
 #include <stdlib.h>
-#ifdef HAVE_UNISTD_H
 #include <unistd.h>
-#else
 #include <values.h>
-#endif
 
 RubberRect::RubberRect(
     Painter* p, Canvas* c, IntCoord x0, IntCoord y0, IntCoord x1, IntCoord y1,
